src/screens/ChatDetailPage/index.js

```diff
@@ -123,7 +124,6 @@ const ChatDetailPage = ({route}) => {
   }, []);
   React.useEffect(() => {
     searchUserMessages(channelClient.channel?.cid);
-    setParticipants(channelClient.channel?.state?.members, dispatch);
   }, [clients.client]);
   const searchUserMessages = async (channelID) => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The call to `setParticipants` has been removed from the `useEffect` hook. If the participants are not set elsewhere after this change, it might lead to issues where the participants' state is not updated correctly.
```diff
-    React.useEffect(() => {
-      searchUserMessages(channelClient.channel?.cid);
-      setParticipants(channelClient.channel?.state?.members, dispatch);
-    }, [clients.client]);
+    React.useEffect(() => {
+      searchUserMessages(channelClient.channel?.cid);
+    }, [clients.client]);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->